reflix/src/components/catalog/Catalog.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react"
+import Movie from "./moviesManager/Movie"
+import Rented from "./rentingManager/Rented"
+import Search from "./searchManager/Search"
+import Budget from "./budgetManager/Budget"
+import "../../styles/catalog.css"
+
+export default function Catalog(props) {
+  const [search, setSearch] = useState("")
+  const [movies, setMovies] = useState([...props.movies])
```

Comment:
No need to duplicate data, if you are sending something down as props you should not store it in state.